test/test_cases/run_3d_rising_bubble.jl

```diff
@@ -35,32 +35,196 @@ function run_3d_rising_bubble(
         parameters = params,
         hyperdiffusivity = FT(100),
     )
+    model_pottemp = Nonhydrostatic3DModel(
+        domain = domain,
+        boundary_conditions = nothing,
+        thermodynamics = PotentialTemperature(),
+        parameters = params,
+        hyperdiffusivity = FT(100),
+    )
 
     # execute differently depending on testing mode
-    if test_mode == :regression
+    if mode == :integration
```

Comment:
Don't need integration. Regression+ validation is fine.

Reply:
Noted and resolved.